integration_tests/e2e/goal/addNote.cy.ts

```diff
@@ -16,6 +16,8 @@ context('Add a note', () => {
     cy.task('stubActionPlansList')
     cy.task('getPrisonerById', 'G6115VJ')
     cy.task('getPrisonerById', 'H4115SD')
+    cy.task('stubGetShortQuestionSetCiagProfile', 'G6115VJ')
```

Comment:
There are quite a few changes like this in this PR - we need to stub the CIAG Induction so that the Overview page displays as the tests expect them to (ie. post-induction)